frontend/src/app/components/Navbar.js

```diff
@@ -100,8 +100,8 @@ class Navbar extends Component {
       >
         <a id="navigation-anchor" href="#navigation-anchor"><span>Navigation Bar</span></a>
         <ul className="nav navbar-nav">
-          {this.props.loggedInUser.role &&
-          this.props.loggedInUser.role.isGovernmentRole &&
+          {this.props.loggedInUser.roles &&
```

Comment:
I'm similarly thinking that this can be simplified since it should be at worst, an empty array.